treeherder/perf/alerts.py

```diff
@@ -15,6 +17,51 @@
                                             detect_changes)
 
 
+def check_confirming_perf_alerts_status():
+    confirming_interval = settings.PERFHERDER_CONFIRMING_INTERVAL
+
+    def traceback_producer_job_type(perf_alert):
+        push = perf_alert.summary.push
+        signature = perf_alert.series_signature
+
+        associated_perf_datum = PerformanceDatum.objects.filter(push=push, signature=signature).first()
+        return associated_perf_datum.job.job_type if associated_perf_datum else None
```

Comment:
@jmaher I don't really understand what's to address here. Could you explain more?

Reply:
I was offering a suggestion for more hacking on this PR, or to open a followup PR

Reply:
> Outside of those comments, I see another change I would like to see. We would now be adding Yet Another Confusing State (YACS) to the sheriffing workflow. I find it difficult already to toggle between untriaged and confirming for the long list of frameworks we support. Can we just assume everything in untriaged is confirming and only moved to confirmed when we have retrigger/backfill data?

Ok, sounds good. This means we must only rename the 'confirming' state to 'confirmed'.

Reply:
There's a distinction between 'untriaged' and 'confirming'.

Reply:
@jmaher We could turn the 'confirming'/'confirmed' states into simple labels per alert. These labels will work just like the star mechanism. This addresses 2 problems: the YACS + the worker not knowing what alerts to check for.
What do you think?

Reply:
interesting thought- I like the ideas of labels better than states- in fact if we can reduce a few states and use labels, that would be a win.  Given the fact that we will use dedicated perf sheriffs for our workflows, I would like to assume the dedicated sheriffs will be mindful of resources and choose the best alerts to work from- then we can focus on changes that support >1 sheriff instead of changes that also support sheriffs who are not trained as much.